Automattic/sensei
# Add email to be sent to student when the teacher replies to a private message

Resolves https://github.com/Automattic/sensei/issues/6482

## Proposed Changes
- Added email that is sent to the student when the teacher replies to a private message

## Testing Instructions

- Enable the email post type
- Run the tool Sensei LMS -> Tools -> Re-Create Emails (Do it even if you have done it before)
- Create a course
- Now to the course as a student and click Contact teacher, send a message (This is the first message, not a reply)
- Now reply to that message from the teacher's account
- Make sure the student has received an email

## Pre-Merge Checklist
